src/app/examples/stock-market/stock-market.effects.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { Actions } from '@ngrx/effects';
+import { cold, getTestScheduler } from 'jasmine-marbles';
+import { of, throwError } from 'rxjs/';
+import {
+  ActionStockMarketRetrieve,
+  ActionStockMarketRetrieveError,
+  ActionStockMarketRetrieveSuccess
+} from './stock-market.actions';
+import { StockMarketEffects, STOCK_MARKET_KEY } from './stock-market.effects';
+import { Stock } from './stock-market.model';
+
+describe('StockMarketEffects', () => {
+  describe('retrieveStock', () => {
```

Comment:
Maybe it would be a good idea to consult the maintainers of @ngrx first. I like skipping the TestBed all together if possible, but the official testing guide does seem to use it. cc @timdeschryver

Reply:
I see, makes sense!

Reply:
Tomasz you really made my mind go poof! I think doing this is perfectly fine and is one of the main reasons why I like Angular and NgRx 😄.

But I'd also like to add that this can't be done for every effect class, from the moment you inject more stuff (like the NgRx store) you'll have to use the Angular TestBed.

Rewriting the NgRx docs is on our schedule, this "hidden gem" is something that can be added imho.

Reply:
@zetsnotdead @timdeschryver so can we agree on saying that we will write tests without `TestBed` as long as it is practical (eg not too many / too complex dependencies to be mocked)?

Reply:
Yea, I'm in favor of it.